### DISCUSSION TITLE
Using STM32F746G-DISCO with LTDC under PlatformIO

### DISCUSSION THREAD

**Discussion by N0ury** [stm32duino/Arduino_Core_STM32#2157] [Q&A]
Hi,
I'm using PlatformIO.
I'd like to use the STM32F746G-DISCO board with the TFT.
I use framework-arduinoststm32 which is native with platformIO.
I can use GPIO, Blue button, Serial... But I would like to use the display.
In libraries dir thre's only these files:
```
CMSIS_DSP		Keyboard		Servo			Wire
CMakeLists.txt		Mouse			SoftwareSerial
EEPROM			RGB_LED_TLC59731	SrcWrapper
IWatchdog		SPI			SubGhz
```
LTDC is not here.
If I copy it from Github I get the error:
`lib/LTDC/src/LTDC.h:97:9: error: 'LTDC_HandleTypeDef' does not name a type; did you mean 'RTC_HandleTypeDef'?
`

Is there a way to solve this?

I have treid to replace all the framework-arduinoststm32 as it is said in some posts (they talk about version number, in my case it's framework-arduinoststm32). But it doesn't work.

It looks like stsm32 Github repo is available in many versions, and it's very hard to understand how to do.
Is there a simple way to use the STM32F746G-DISCO board under platformIO?

I have tried also with Arduino IDE. It is as difficult to understand as with PlatformIO.

Thanks to everyone that can help.

**Comment by N0ury** [comment]
Salut Frédéric, je crois comprendre que tu parles français. Si c'est le cas, ce sera nettement plus simple pour moi.
Mon anglais est très limité.

**Reply by fpistm** [reply]
Bonjour effectivement.
Concernant le point LTDC, nous ne supportons pas PIO. Ni LTDC d'ailleur dans Arduino.
Donc la librairie dont tu parles je ne la connais pas.
Je peux juste conseiller d'activer le module HAL LTDC : `HAL_LTDC_MODULE_ENABLED`

**Reply by fpistm** [reply]
Duplicate of #1532 and #1530
